configure.ac

```diff
@@ -980,6 +980,15 @@ if test "${enable_sm}" = "yes"; then
 	esac
 fi
 
+case "${host}" in
+       *-winnt*|*-cygwin*)
+                       win_abs_top_srcdir="$(cygpath --windows $(realpath ${srcdir}))"
```

Comment:
This assumes that we're always using the cygwin shell. This may change with #2754 or #3251.